### ISSUE TITLE
geom_qq_line() issues warning since #4866

### ISSUE THREAD

**Issue by capnrefsmmat** [tidyverse/ggplot2#5010]
Consider the first example on the [geom_qq documentation page](https://ggplot2.tidyverse.org/reference/geom_qq.html):

```r
df <- data.frame(y = rt(200, df = 5))
p <- ggplot(df, aes(sample = y))
p + stat_qq() + stat_qq_line()
```

In the released version of ggplot2, this issues no warning. On the `v3.4.0-rc` branch, this produces the warning

```
Warning message:
The following aesthetics were dropped during statistical transformation: sample
ℹ This can happen when ggplot fails to infer the correct grouping structure in the data.
ℹ Did you forget to specify a `group` aesthetic or to convert a numerical variable into a
  factor? 
```

Removing `stat_qq_line()` eliminates the warning. I believe this is due to #4866. I believe the warning is spurious, since no grouping aesthetics are necessary here.

**Comment by clauswilke** [comment]
Thanks for pointing this out.

@thomasp85 We should fix this before the 3.4.0 release. It's a simple `dropped_aes = "sample"` inserted somewhere here: https://github.com/tidyverse/ggplot2/blob/a58b48c961cb391b8646bf072b6620a0c9f3d999/R/stat-qq-line.R#L47-L49

**Comment by thomasp85** [comment]
Thanks - there have been one more of these already with `weight` and `stat_count()`... will fix this in the rc-branch
